pkg/cmd/server/admin/create_client.go

```diff
@@ -177,3 +177,20 @@ func (o CreateClientOptions) CreateClientFolder() error {
 
 	return nil
 }
+
+// readFiles returns a byte array containing the contents of all the given filenames,
```

Comment:
who uses this?

Reply:
Other commands from the same package, but it might die with all them being removed.